gdx/src/com/badlogic/gdx/graphics/GLTexture.java

```diff
@@ -164,6 +170,62 @@ public void setFilter (TextureFilter minFilter, TextureFilter magFilter) {
 		Gdx.gl.glTexParameteri(glTarget, GL20.GL_TEXTURE_MAG_FILTER, magFilter.getGLEnum());
 	}
 
+	/** Sets the anisotropic filter level for the texture. Assumes the texture is bound and active!
+	 * @param level The desired level of filtering. The maximum level supported by the device up to this value will be used.
+	 * @return The actual level set, which may be lower than the provided value due to device limitations. */
+	public float unsafeSetAnisotropicFilter (float level){
+		return unsafeSetAnisotropicFilter(level, false);
+	}
+
+	/** Sets the anisotropic filter level for the texture. Assumes the texture is bound and active!
+	 * @param level The desired level of filtering. The maximum level supported by the device up to this value will be used.
+	 * @param force True to always set the value, even if it is the same as the current values.
+	 * @return The actual level set, which may be lower than the provided value due to device limitations. */
+	public float unsafeSetAnisotropicFilter (float level, boolean force){

```

Comment:
Inconsitent method formatting. Should have space after method name, and after the final param parenthesis.